hadoop-project/pom.xml

```diff
@@ -66,6 +66,7 @@
 
     <!-- jersey version -->
     <jersey.version>1.19</jersey.version>
+    <jersey2.version>2.28</jersey2.version>
```

Comment:
Can we bump it to 2.29 to support JDK 11?
http://blog.supol.cz/?p=144
Jersey 2.29 supports JDK11

Reply:
Thanks. Updated.